miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/StorageLocationRestController.java

```diff
@@ -76,15 +77,15 @@
     return doSave(dto);
   }
 
-  private StorageLocationDto doSave(StorageLocationDto dto) {
+  private StorageLocationDto doSave(StorageLocationDto dto) throws IOException {
     StorageLocation freezer = Dtos.to(dto);
     long savedId = storageLocationService.saveFreezer(freezer);
     StorageLocation saved = storageLocationService.get(savedId);
     return Dtos.asDto(saved, true, true);
   }
 
   @PostMapping(value = "/freezers/{id}/shelves")
-  public @ResponseBody StorageLocationDto addFreezerShelf(@PathVariable(name = "id", required = true) long id) {
+  public @ResponseBody StorageLocationDto addFreezerShelf(@PathVariable(name = "id", required = true) long id) throws IOException {
     StorageLocation freezer = getFreezer(id);
     int lastShelf = freezer.getChildLocations().stream().filter(loc -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)